content/scripts/build.js

```diff
@@ -222,17 +222,23 @@ function extractLocale(source, folder) {
   return locale;
 }
 
-function getMetadata(source, folder) {
-  const metadataRaw = fs.readFileSync(path.join(folder, "index.yaml"));
-  const metadata = yaml.safeLoad(metadataRaw);
-  const wikiHistoryPath = path.join(folder, "wikihistory.json");
-  if (fs.existsSync(wikiHistoryPath)) {
-    const wikiMetadataRaw = fs.readFileSync(wikiHistoryPath);
-    const wikiMetadata = yaml.safeLoad(wikiMetadataRaw);
-    metadata.modified = wikiMetadata.modified;
+function getContent(source, folder, appendWikiHistory = false) {
```

Comment:
❤️  the `appendWikiHistory = false`, since most of the time we don't need the `modified` attribute. 👍